### PR TITLE
Fix translation, fix reinstall table already exists, fix deprecated datetime, Add german translation

### PR DESCRIPTION
**Pull request by invisiblemarcel**
These changes are pulled separately because of the major changes.
- I always needed to drop the tables manually when I uninstalled the plugin, so I added 2 checks to avoid that.
- It was not enough to copy a .po File for translation, I needed to compile them so I add a note in the Readme and compiled all lang files and added them to the repo (I know binary files in git are crap but it's much easier for the users)
- German translation added
- The table column type datetime is deprecated so I changed it to timestamp, before you always needed to run a command displayed in the UI to migrate it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/locales/*.mo
```
Comment (edgardmessias) [inline .gitignore:1]: Please, don't upload the `.mo` files, because they are binaries

File: generic_examples/zitadel.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+- Go to Zitadel and login as admin
```
Comment (edgardmessias) [inline generic_examples/zitadel.md:1]: This file will be conflicted with another PR

File: hook.php [hunk 3]
```diff
@@ -235,8 +235,8 @@ function plugin_singlesignon_install() {
                   `split_name`                 tinyint(1) NOT NULL DEFAULT '0',
                   `is_deleted`                 tinyint(1) NOT NULL default '0',
                   `comment`                    text COLLATE utf8_unicode_ci,
-                  `date_mod`                   datetime DEFAULT NULL,
```
Comment (edgardmessias) [inline hook.php:238]: Why this change?

**Review by edgardmessias — CHANGES_REQUESTED** [review]
Please, could you rebase your PR?
 My suggestion: split the PR for each type of change

File: hook.php (hunk 3 above)
Comment (invisiblemarcel) [reply hook.php:238]: Because of this message after activating the plugin:

grafik

I always need to run the command (`php bin/console migration:timestamps`) described there. In the database schema I saw that this command change the type of the 2 columns to datetime. So I decided to change the type directly in the create table. Maybe it depends on the glpi version. I'm working on version 10.0.14.
